core/domain/takeout_service.py

```diff
@@ -101,39 +103,18 @@ def export_data_for_user(user_id: str) -> takeout_domain.TakeoutData:
         final_name = ('_').join([x.lower() for x in split_name])
         exported_data[final_name] = exported_model_data
 
-    # Separate out images. We store the images that need to be separated here
-    # as a dictionary mapping tuples to strings. The tuple value indicates the
-    # "path" to take to the image in the user's data dictionary, and the string
-    # indicates the filename that the exported image will be saved to.
-    replacement_instructions = [
-        takeout_domain.TakeoutImageReplacementInstruction(
-            ('user_settings', 'profile_picture_data_url'),
-            'user_settings_profile_picture.png',
-            'profile_picture_filename'
-        )
-    ]
+    # Separate out images. We store the images that need to be separated here,
+    # as the images are now stored in GCS we will fetch them and return them.
     takeout_image_files: List[takeout_domain.TakeoutImage] = []
-    for replacement_instruction in replacement_instructions:
-        dictionary_path = replacement_instruction.dictionary_path
-        replacement_filename = replacement_instruction.export_filename
-        replacement_key = replacement_instruction.new_key
-
-        # Move pointer to the position indicated by the tuple.
-        pointer = exported_data
-        for key in dictionary_path[:-1]:
-            pointer = pointer[key]
-
-        # Swap out data with replacement filename.
-        image_key = dictionary_path[-1]
-        image_data = pointer[image_key]
-        if image_data is not None:
-            # Ruling out the possibility of Any for mypy type checking.
-            assert isinstance(image_data, str)
+    if user_settings is not None:
```

Comment:
What about the webp image?

Reply:
I think we previously decided that we would not require to output the webp images that is why I did not added them.

Reply:
If we store user data in whatever form, then we need to export it, right?

Reply:
Updated!